news/2020-05-08-mappers-report-april.md

```diff
@@ -0,0 +1,193 @@
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+The Mappers' Showcase will bring to you Mappers from every game mode which not everyone might know. They are hand-selected and a hidden gem for their game mode. 
+This month will feature osu!taiko and the selectors were [radar](https://osu.ppy.sh/users/7131099) and [Capu](https://osu.ppy.sh/users/2474015).
+
+### Idealism
+
+**[Idealism](https://osu.ppy.sh/users/3869519)** is a osu!taiko mapper (as well as an osu! top player) from Brazil who has been around in the community for a while now, though he definitely deserves more credit for his work in the mapping scene! His maps blend creativity with smooth and interesting patterning which will leave a good taste in the mouths of mappers and players alike.
+
+A map of his that truly shines in many people's eyes is his semi-recently ranked rendition of [Cyte - Command Prompt](https://osu.ppy.sh/beatmapsets/1068145#taiko/2236237). On the top difficulty (mind bogglingly named "C:\Program Files\Difficulties\434f525255505445442046494c45.exe"), Idealism proves that even when mapping genres out of his comfort zone he can still shine. 
+
+Some of his other notable and equally as great maps would be his renditions of [Blue on Blue (TamorlaM Btlg Remix)](https://osu.ppy.sh/beatmapsets/965135#taiko/2020395) and [White Eternity](https://osu.ppy.sh/beatmapsets/992443#taiko/2075446), where he shows off his addiction to eroge music through his solid mapping! We hope to see even more of his maps coming into the ranked section in the rest of 2020, as his potential to shine even brighter clearly shows!
+
+### Yuemiao
+
+Recently making her first appearance in the ranked section with her take on [Mizutama by YUC'e and Nor](https://osu.ppy.sh/beatmapsets/1009290#taiko/2112927),  **[Yuemiao](https://osu.ppy.sh/users/4493348)** shows us that she definitely knows what she's doing.
+
+Executing many different snaps and rhythms perfectly, with clean consistency throughout her difficulties, making this map definitely one of the most interesting ones in the ranked section as of late.
+Adding her own little rhythms with an intuitive yet interesting kat placement, really shows how good osu!taiko can sound and how it's possible to enhance the song that is mapped when someone knows their craft!
+
+Her ability to also map faster and more chaotic songs just goes to show that she can handle multiple genres and definitely can create something clean for everyone's taste. Try it yourself with this well polished unranked set of [IZANA - t+pazolite vs. P*Light](https://osu.ppy.sh/beatmapsets/1100797#taiko/2299870), which also includes very smooth SV sections, making this pleasant to the eye as well.
+Yuemiao definitely deserves more credit, as she is quite unknown and surprised many with her first ranked map. The current 28 ratings on her Mizatuma consist of 27(!) 10* votes. If ratings could talk, we wouldn't have to phrase this.
```

Comment:
the exclamation point looks a little colloquial, could maybe emphasise in a more literary way like this?
```suggestion
Yuemiao definitely deserves more credit, as she is quite unknown and surprised many with her first ranked map. From the current 28 ratings on her Mizatuma set, 27 (*twenty-seven!*) are 10* votes, which speaks for itself.
```